src/gettsim/germany/einnahmen/renten/inputs.py

```diff
@@ -14,6 +14,12 @@ def sonstige_private_vorsorge_m() -> float:
     """
 
 
+@policy_input()
+def alter_beginn_leistungsbezug_sonstige_private_vorsorge() -> int:
```

Comment:
This only matters for Einkommensteuer, right? Shouldn't it be somewhere in that namespace?